src/gwt/src/org/rstudio/studio/client/panmirror/outline/PanmirrorOutline.css

```diff
@@ -0,0 +1,4 @@
+@charset "UTF-8";
+
+.outline {
+}
```

Comment:
Do we still need this file?

Reply:
Nope. Removed here: https://github.com/rstudio/rstudio/commit/36e52c2b69ccff880db14de0b5f012986a22345a